torchbearer/magics.py

```diff
@@ -17,7 +17,7 @@ def torchbearer(line):
     import IPython.core.magic
     torchbearer = IPython.core.magic.register_line_magic(torchbearer)
     set_notebook(True)
-except NameError:
+except NameError or ModuleNotFoundError:
```

Comment:
@ethanwharris Do you think we should just catch generic exception here?

Reply:
Nope, in theory we should always avoid catching Exception